src/pages/overlay/App.tsx

```diff
@@ -54,6 +56,8 @@ export default function App() {
         onTouchMove={interacted}
         onKeyDown={interacted}
         onMouseLeave={sleep}
+        onFocus={() => stayAwake()}
+        onBlur={() => resumeTimer(timeout)}
```

Comment:
Just want to confirm that this isn't going to interfere with the automatic sleeping, clicking on buttons in the UI won't cause onFocus to fire?